video-store/part-1-submit.md

```diff
@@ -0,0 +1,32 @@
+# Part 1 Submit: Video Store API
+
+## Submit API for Feedback
+
+<!-- prettier-ignore-start -->
+### !challenge
+
+* type: short-answer
+* id: 0f51eeda-d431-40a6-94fe-a07d2fd1af39
+* title: Pull Request for Feedback
+
+##### !question
+
+Make a pull request against the [Video Store API repo](https://github.com/AdaGold/retro-video-store). Place the URL of the pull request here. Review instructions in the [How to Submit](../ada-project-practices/how-to-submit.md) repo if needed.
+
+##### !end-question
+
+##### !placeholder
+The URL to your project repo on GitHub: https://github.com/<your-username>/<project-name>
+##### !end-placeholder
+
+##### !answer
+
+/.+/
```

Comment:
Just a suggestion to help make sure it's a valid PR.

```suggestion
/^https:\/\/github\.com\/[^\/]+\/[^\/]+\/pull\/\d?\d[\/]?/?/
```

Reply:
Thank you!!!